Sources/Currency/CurrencyValue+StringRepresentation.swift

```diff
@@ -61,6 +61,7 @@ extension String.StringInterpolation {
   ///   - value: The value to localize. If the value is `nil`, then the `nilDescription` will be used.
   ///   - locale: The Locale to localize the value for. The default is `.current`, ig. the runtime environment's Locale.
   ///   - nilDescription: The optional description to use when the `value` is `nil`.
+  @_documentation(visibility: private)
```

Comment:
If we are exposing the function why are we hiding its documentation?

Reply:
Because of quirks with DocC and its generation.

We show this exists for people in documentation, and have symbol docs for code completion, but generally these methods aren’t able to be accessed directly - so there’s no point in listing it in the main side bar

Reply:
Makes sense. Personally do not think people will use the interpolation methods since they are hard to "discover"